odxtools/diaglayers/protocol.py

```diff
@@ -24,14 +26,26 @@ class Protocol(HierarchyElement):
     def protocol_raw(self) -> ProtocolRaw:
         return cast(ProtocolRaw, self.diag_layer_raw)
 
+    @property
+    def comparam_spec_ref(self) -> OdxLinkRef:
+        return self.protocol_raw.comparam_spec_ref
+
     @property
     def comparam_spec(self) -> ComparamSpec:
         return self.protocol_raw.comparam_spec
 
+    @property
+    def prot_stack_snref(self) -> str | None:
+        return self.protocol_raw.prot_stack_snref
+
     @property
     def prot_stack(self) -> ProtStack | None:
         return self.protocol_raw.prot_stack
 
+    @property
+    def parent_refs(self) -> list[ParentRef]:
```

Comment:
Really needed after the database links has been resolved?

Reply:
the "logical view" is supposed to expose *all* properties in their "final" form, i.e., if something needs to be computed -- like value inheritance -- non-raw diaglayers should expose the processed sets of objects, if there's no need for processing -- e.g., when specifying the parent layers (`parent_refs`) -- it should just forward these properties from the respective raw layers. (At least that was my philosophy when writing this code, I probably made a few mistakes besides the ones fixed by this PR...)

> not just their refs

be aware that the parent refs are a bit quirky here: they do reference resolution internally, i.e. they provide access to the respective "full" parent layer object via their `.layer` attribute.